.git-blame-ignore-revs

```diff
@@ -0,0 +1,18 @@
+# git-blame ignored revisions
+# To configure, run
+#   git config blame.ignoreRevsFile .git-blame-ignore-revs
+# Requires Git > 2.23
+# See https://git-scm.com/docs/git-blame#Documentation/git-blame.txt---ignore-revs-fileltfilegt
+
+# Reformatting PR was primarily done in #347
+
+# Initial black reformatting
+40696f941ca2a2b30938fbd2bcb8e197a46024d5
+# Initial isort reformatting
+6732b719834fc3a7d72dfd78f684f066a1fcf11f
```

Comment:
Not sure if this is worth adding as the diff is pretty small: https://github.com/wagtail/bakerydemo/commit/6732b719834fc3a7d72dfd78f684f066a1fcf11f